test/javascripts/acceptance/question-answer-test.js

```diff
@@ -584,6 +586,30 @@ acceptance("Discourse Question Answer - logged in user", function (needs) {
     );
   });
 
```

Comment:
The description here doesn’t really reflect what the assertions are doing since there is no voting action taking place in the tests.

Reply:
I am not sure if we can simulate a click even on disabled button. I am seeing this error `Can not `click` disabled [object HTMLButtonElement]`.

Reply:
We don't need this anymore as we are not replying on client side button enabling/disabling.